### DISCUSSION TITLE
HACS version mixed signals!?

### DISCUSSION THREAD

**Discussion by mtojay** [ExperienceLovelace/ha-floorplan#374] [Bugs] [answered]
I have ha-floorplan installed through HACS. Latest Version (1.0.39):

![grafik](https://github.com/ExperienceLovelace/ha-floorplan/assets/30782895/739e2f0f-cf40-4d85-90e9-ab515fe2f88a)

but i just realized that my info log shows i would be running 1.0.37.  Now i am not quite sure if this is just a display issue or if i am really working on an older version.

![grafik](https://github.com/ExperienceLovelace/ha-floorplan/assets/30782895/73cec3a4-91b5-45c6-bd0a-5c57882652fe)

**Comment by exetico** [comment, marked as answer]
That's on my to-do. I've already fixed it in the main branch, but I didn't really want to push a release only for a release-version bump in the console 😏😁.

The explainer is that I need to bump the version in a specific file before a release, and I clearly forget that, now and then 😁.

**Reply by exetico** [reply]
I remember bumped it as part of .40. I'll think of a smarter way down the road.

Thank you for bringing that on the table 😊.
